bin/deliver_poste_messages

```diff
@@ -270,18 +271,21 @@ def main()
     until queue.empty?
       next unless delivery = queue.pop(true) rescue nil
 
+      sent_at = DateTime.now
+
       begin
         deliverer.send delivery
-      rescue Net::SMTPSyntaxError => e
-        puts "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{e.message.to_s.strip}'"
-      rescue Net::SMTPFatalError => e
+      rescue Net::SMTPSyntaxError, Net::SMTPFatalError => e
         puts "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{e.message.to_s.strip}'"
```

Comment:
Yeah, current behavior is to not send it and set the sent_at date the DateTime.now (so it doesn't get retried).

looking at http://ruby-doc.org/stdlib-2.0.0/libdoc/net/smtp/rdoc/Net/SMTPFatalError.html http://ruby-doc.org/stdlib-2.0.0/libdoc/net/smtp/rdoc/Net/SMTPSyntaxError.html  https://tools.ietf.org/html/rfc5321#section-4.2.1 I think we shouldn't retry these.

````
5yz  Permanent Negative Completion reply
      The command was not accepted and the requested action did not
      occur.  The SMTP client SHOULD NOT repeat the exact request (in
      the same sequence).
````